### PR TITLE
Improve readability of Electric documentation

### PR DESCRIPTION
**Pull request by rdai10**

### CODE REVIEW

**Comment by jbalsas** [comment]
@Robert-Frampton, @blzaugg, does this look good? Could we merge or close? :)

File: src/pages/docs/develop/errorHandling.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight: 2
 
 ## Error Handling
 
-It's important to understand how to handle errors when using Senna.js. The three error handling options in Senna.js are Invalid Status, Request Error, and Timeout.
+It's important to understand how to handle errors when using Senna.js. The three error handling options in Senna.js are `Invalid Status`, `Request Error`, and `Timeout`.
```
Comment (blzaugg) [inline src/pages/docs/develop/errorHandling.md:14]: - [x] **Bold** these instead, as they are not `code`.
- [x] Or use the property names instead, eg. `invalidStatus`.

File: src/pages/docs/develop/routes.md [hunk 2]
```diff
@@ -41,6 +41,6 @@ var route = new senna.Route(resolvePath, function() {
 ```
 
 
-From the handler callback you can update parts of your page, though it is hard to grow your application from a simple callback like this. For that reason, Senna.js provides `senna.Screen`. A screen is nothing more than a special type of route handler that provides an asynchronous lifecycle. The following section will detail how `senna.Screen` can help you manage your surfaces.
+From the handler callback you can update parts of your page, though it is hard to grow your application from a simple callback like this. For that reason, Senna.js provides `senna.Screen`. A screen is nothing more than a special type of route handler that provides an asynchronous lifecycle. You can also jump to [Screens](/docs/fullyFunctional/screens.html) to learn more about how `senna.Screen` can help you manage your surfaces.
```
Comment (blzaugg) [inline src/pages/docs/develop/routes.md:45]: - [x] How about?
... provides an asynchronous lifecycle. [Learn more about how Screens can help you manage your surfaces](/docs/fullyFunctional/screens.html).

File: src/pages/docs/fullyFunctional/controllingSurfaces.md [hunk 3]
```diff
@@ -10,21 +10,23 @@ weight: 2
 
 ## How do you make sure Senna does exactly what you want?
 
-By default, Senna is setup to route every link to `senna.HTMLScreen`- the out-of-the-box screen that senna provides to handle navigation so you don't have to change anything in your existing back end.
+By default, Senna is setup to route every link to `senna.HTMLScreen` - the out-of-the-box screen that senna provides to handle navigation so you don't have to change anything in your existing back end.
```
Comment (blzaugg) [inline src/pages/docs/fullyFunctional/controllingSurfaces.md:14]: - [x] Use em dash `–`. Remove spaces around it.
- [x] Add another at `handle navigation–so you don't`

File: src/pages/docs/fullyFunctional/controllingSurfaces.md [hunk 4]
```diff
@@ -10,21 +10,23 @@ weight: 2
...
 
 For most situations, this will be the ideal use of Senna because it is smart enough to know how to paint your surfaces with the new information when you navigate.
 
-But what if you want Senna to only be active during certain navigations? We make it easy for you to control what screens of your app interact with Senna and which ones do not. We call this [Controlling Data Attributes](#taking-control).
+But what if you want Senna to only be active during certain navigations? We make it easy for you to control what screens of your app will interact with Senna and what will not not. We call this [Controlling Data Attributes](#taking-control).
```
Comment (blzaugg) [inline src/pages/docs/fullyFunctional/controllingSurfaces.md:18]: - [x] Remove extra "not". `what will not not`

File: src/pages/docs/fullyFunctional/screens.md [hunk 5]
```diff
@@ -12,15 +12,15 @@ weight: 1
 
 Senna.js provides a special type of route handler, or interface, called `senna.Screen`. This in itself is not an active class, rather, it merely defines the screen interface methods and demonstrates how to use the API. The real magic happens when we extend `senna.Screen`.
 
-We knew this might be a little tricky at first, so we creating our own extention: `senna.HTMLScreen`- the default screen we created to make Senna.js ready to work straight out of the box.
+We knew this might be a little tricky at first, so we created our own extention: `senna.HTMLScreen` - the default screen for Senna.js straight out of the box.
```
Comment (blzaugg) [inline src/pages/docs/fullyFunctional/screens.md:16]: - [x] Use em dash `–`. Remove spaces around it.
- [x] Add another em dash at: `Senna.js–straight out of the box.`
- [ ] Look for more uses of dash, that should be em dashes.

File: src/pages/docs/gettingStarted/howdoesitwork.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ When the user clicks a link to another page of your app, Senna.js begins to work
 
 ## 3. Request is made for matching screen
 
-The request is made for the new page but instead of pulling down a completely new package of elements from the server, Senna.js utilizes the cached resources that are redundant to both the pages.
+The request is made for the new page but instead of pulling down a completely new package of elements from the server, Senna.js utilizes the cached resources that are shared for both pages.
```
Comment (blzaugg) [inline src/pages/docs/gettingStarted/howdoesitwork.md:30]: - [x] Reword to: "shared across both pages".

File: src/pages/docs/gettingStarted/theBuildingblocks.md [hunk 7]
```diff
@@ -102,7 +102,7 @@ By using History API, you can manipulate the browser history to use the browser'
 
 ## Cacheable Screens:
 
-Once you load a certain surface, this content is cached in memory and will be retrieved later on without any additional request to the server.
+Once you load a certain surface, this content is cached in memory and will be retrieved later without any additional request to the server.
```
Comment (blzaugg) [inline src/pages/docs/gettingStarted/theBuildingblocks.md:105]: - [x] Consider rewording to: "and will be retrieved on subsequent loads, without any additional"

**Comment by blzaugg** [comment]
@jbalsas, I'll make the changes I've commented on.

**Comment by blzaugg** [comment]
@Robert-Frampton, go ahead and close this. I'll have @rdai10, make the corrections.

**Pull request closed without merging**